rsmas_logging.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+class rsmas_logger():
+
+    format = "%(levelname)s - %(message)s"
+    console_handler, file_handler = None, None
+    logfile_name = None
+    logger = logging.getLogger()
+
+    def __init__(self, console=True, file=None):
+
+        self.logfile_name = file
+
+        self.logger.setLevel(logging.DEBUG)
+        self.set_format(self.format)
+
+
+    def setup_filehandler(self, formatter):
+        file_handler = logging.FileHandler(self.logfile_name, 'a+', encoding=None)
+        file_handler.setLevel(logging.INFO)
+        file_handler.setFormatter(formatter)
+        return file_handler
+
+    def setup_consolehandler(self, formatter):
+        console_handler = logging.StreamHandler()
+        console_handler.setLevel(logging.INFO)
```

Comment:
Will you ever want `logging.DEBUG` for the console?

Reply:
Good point. The nice thing about the logging module is that it logs everything at or higher than the given level.